pkg/framework/guest.go

```diff
@@ -238,16 +238,20 @@ func (g *Guest) WaitForNodesUp(numberOfNodes int) error {
 		}
 
 		if len(nodes.Items) != numberOfNodes {
-			return microerror.Maskf(waitError, "worker nodes are still not found")
+			return microerror.Maskf(waitError, "found %d k8s nodes but want %d", len(nodes.Items), numberOfNodes)
 		}
 
+		var notReadyCnt int
```

Comment:
Is that name on purpose? I would go with `notReadyCount`.

Reply:
It's on purpose. It's pretty common abbreviation AFAIK. If you want provide suggestions and I change it.